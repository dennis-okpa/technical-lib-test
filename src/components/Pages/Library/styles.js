import styled from 'styled-components'

export const Container = styled.div`
  padding: 32px;
`

export const Header = styled.header``

export const Line = styled.div`
  width: 100%;
  border-top: 2px solid #d3d3d3;
  margin: 30px 0 24px;
`
