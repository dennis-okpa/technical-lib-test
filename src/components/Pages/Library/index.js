import React from 'react'
import { Container, Header, Line } from './styles'
import Gallery from '../../UI/Gallery/index'

function Library(props) {
  return (
    <Container>
      <Header>
        <h1>Library</h1>
      </Header>
      <Line />
      <Gallery />
    </Container>
  )
}
export default Library
