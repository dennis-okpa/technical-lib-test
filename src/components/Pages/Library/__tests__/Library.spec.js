import Library from '../index'
import { render } from '@testing-library/react'
import React from 'react'

describe('The Library component', () => {
  it('should match snapshot', () => {
    const { container } = render(<Library />)
    expect(container).toMatchSnapshot()
  })
})
