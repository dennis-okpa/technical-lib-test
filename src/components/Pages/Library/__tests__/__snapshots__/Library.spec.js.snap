// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Library component should match snapshot 1`] = `
<div>
  <div
    class="sc-bdvvtL cydklE"
  >
    <header
      class="sc-gsDKAQ"
    >
      <h1>
        Library
      </h1>
    </header>
    <div
      class="sc-dkPtRN hpEdkZ"
    />
  </div>
</div>
`;
